py/h2o_wave/server.py

```diff
@@ -73,6 +73,21 @@ def __init__(self, username: str, subject: str, access_token: str, refresh_token
         """The access token of the user."""
         self.refresh_token = refresh_token
         """The refresh token of the user."""
+        self._session_id = session_id
+        """Session identifier. Do not access, internal use only."""
```

Comment:
nit: I don't know how the doc generation works so I just mention that this is internal attribute  so it should not (IMHO) appear in the documentation.

Reply:
These are ignored during docs build. The docstring is for people that like to explore code manually, in which case I guess it's fine.